commons/build.gradle

```diff
@@ -28,6 +28,6 @@ repositories {
 }
 
 dependencies {
-    compile('org.hibernate.javax.persistence:hibernate-jpa-2.1-api:1.0.0.Final')
+    compile('org.springframework.boot:spring-boot-starter-data-jpa:1.5.3.RELEASE')
```

Comment:
Why do we need spring here? Commons should not know anything about spring, Hibernate should be enough here.